pkg/yurttunnel/server/cmd.go

```diff
@@ -125,6 +136,7 @@ func (o *YurttunnelServerOptions) validate() error {
 func (o *YurttunnelServerOptions) complete() error {
 	o.serverAgentAddr = fmt.Sprintf("%s:%d", o.bindAddr, o.serverAgentPort)
 	o.serverMasterAddr = fmt.Sprintf("%s:%d", o.bindAddr, o.serverMasterPort)
+	o.serverMasterInsecureAddr = fmt.Sprintf("%s:%d", o.bindAddr, o.serverMasterInsecurePort)
 	klog.Infof("server will accept %s requests at: %s, "+
 		"server will accept master requests at: %s",
 		projectinfo.GetAgentName(), o.serverAgentAddr, o.serverMasterAddr)
```

Comment:
add serverMasterInsecureAddr info in the log can make help

Reply:
add serverMasterInsecureAddr info to log